clusterloader2/pkg/measurement/common/etcd_metrics.go

```diff
@@ -196,6 +196,8 @@ func (e *etcdMetricsMeasurement) getEtcdMetrics(host string, provider provider.P
 	cmd := fmt.Sprintf("curl http://localhost:%d/metrics", port)
 	if samples, err := e.sshEtcdMetrics(cmd, host, provider); err == nil {
 		return samples, nil
+	} else {
```

Comment:
nit - above you error.

So instead of adding else, just log it unconditionally here (it's semantically equivalent)